k8s/stset/lrp_to_statefulset.go

```diff
@@ -261,6 +260,14 @@ func getContainerResources(cpuWeight uint8, memoryMB, diskMB int64) corev1.Resou
 	}
 }
 
+func NewMebibyteQuantity(miB int64) resource.Quantity {
+	memory := resource.Quantity{
+		Format: resource.BinarySI,
+	}
+	memory.SetScaled(miB*1024*1024, 0)
```

Comment:
There is a `Set()` method which would avoid passing the zero as the second arg here

Reply:
Changed to use Set instead of SetScaled